### PR TITLE
Icons now visible in light mode and new twitter logo throughout docs

### PR DESCRIPTION
**Pull request by Akshat-bhatia73**
## What kind of change does this PR introduce?

Bug fix, docs update

## What is the current behavior?
https://github.com/supabase/supabase/issues/16801

## What is the new behavior?
Now all the icons properly visible in light mode.
Also added an svg file for twitter icons.
Screenshot 2023-08-25 121820
Screenshot 2023-08-25 121844
Screenshot 2023-08-25 121858

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 1]
```diff
@@ -435,6 +437,7 @@ export const PhoneLoginsItems = [
     linkDescription:
       'Vonage is a communication platform as a service (CPaaS) provider for consumers and businesses.',
     isDarkMode: true,
+    hasLightIcon: true,
```
Comment (fsansalvadore) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:440]: This vonage icon still seems to not show correctly in light mode.
Can you double check this one?

https://docs-git-fork-akshat-bhatia73-fix-twitter-logo-supabase.vercel.app/docs/guides/auth

Screenshot 2023-08-25 at 10 26 58

**Review by fsansalvadore — CHANGES_REQUESTED** [review]

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts (hunk 1 above)
Comment (Akshat-bhatia73) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:440]: Sure, I'll check.

Comment (Akshat-bhatia73) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:440]: I have made the change so please verify if the issue has resolved or not.

Comment (fsansalvadore) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:440]: Awesome, that looks good now 👍

**Comment by fsansalvadore** [comment]
Looks good @Akshat-bhatia73 

Prettier is failing in the `apps/docs/pages/guides/getting-started.mdx` page.
Running `npm run format` in the root directory should fix it, otherwise use [Prettier formatter](https://www.alphr.com/use-prettier-vs-code/) if using vscode.

**Review by fsansalvadore — APPROVED** [review]

**Pull request merged**